modules/local/star_align.nf

```diff
@@ -54,12 +54,18 @@ process STAR_ALIGN {
     // separate forward from reverse pairs
     def (forward, reverse) = reads.collate(2).transpose()
     """
+    if [[ $whitelist == *.gz ]]; then
+        gzip -cdf $whitelist > whitelist.txt
+    else
+        cp $whitelist whitelist.txt
```

Comment:
This would fail if the filename is already "whitelist.txt":
```
cp: 'whitelist.txt' and 'whitelist.txt' are the same file
```

Maybe something like 
```
${whitelist}.uncompressed.txt
```

could work

Reply:
Good call, fixed